helm: update the sidecar mentions in the Chart Description

humblec opened the pull request:
RBD chart description didnt have `resizer` in it and this commit
add the same.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Madhu-1 commented on charts/ceph-csi-rbd/Chart.yaml:
```diff
@@ -2,7 +2,7 @@
 appVersion: canary
 description: "Container Storage Interface (CSI) driver,
-provisioner, snapshotter, and attacher for Ceph RBD"
+provisioner, snapshotter, resizer and attacher for Ceph RBD"
```
same change is required in cephfs chart also.

humblec replied:
@Madhu-1 I have done that on attacher removal PR # https://github.com/ceph/ceph-csi/pull/3149/commits/d86508274c75174787abf7d83e2c5056b77a5db1

Madhu-1 replied:
@humblec can you add that to this PR as it makes sense to make the same changes in the similar PR?

humblec replied:
@Madhu-1 Done.

Madhu-1 replied:
@humblec i dont see the changes yet, please check again.

humblec replied:
@Madhu-1 you meant to pull that changes here ? ok.. I can do that as well.

Madhu-1 replied:
@humblec any specific reason to remove cephfs change from this PR?

humblec replied:
@Madhu-1 nothing has to be changed there https://github.com/ceph/ceph-csi/blob/devel/charts/ceph-csi-cephfs/Chart.yaml ?

Madhu-1 replied:
I missed the mentioned PR already merged.

humblec commented:
@Madhu-1 done.. ptal.

Madhu-1 reviewed (commented):
@humblec please squash the commits and use doc/helm in the commit subject.

humblec commented:
@Madhu-1 done. :+1:

humblec commented:
/retest ci/centos/mini-e2e/k8s-1.22

humblec commented:
/retest ci/centos/mini-e2e/k8s-1.21

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.22

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.22

pkalever commented:
@Mergifyio unqueue

pkalever commented:
This PR is blocking the merging train since at least 15 hours now, hence unqueued it

pkalever commented:
@Mergifyio requeue

pkalever commented:
@Mergifyio refresh

humblec commented:
@Mergifyio refresh

humblec commented:
@Mergifyio refresh

humblec commented:
5 days in the queue still not merged :) !!

humblec commented:
@Mergifyio refresh

humblec commented:
@Mergifyio rebase

humblec commented:
@Mergifyio rebase

humblec committed:
helm: update the sidecar mentions in the Chart Description

CephFS chart description didnt have `resizer` in it
and this commit add the same.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.23

humblec commented:
@Madhu-1 @yati1998 just rebased to master.  ptal.

humblec commented:
/retest ci/centos/mini-e2e-helm/k8s-1.23

Rakshith-R commented:
it's just description text change, no need to run ci 
@mergifyio rebase

Rakshith-R commented:
/retest ci/centos/mini-e2e-helm/k8s-1.22
